ts/input/tex.ts

```diff
@@ -217,6 +218,15 @@ export class TeX<N, T, D> extends AbstractInputJax<N, T, D> {
    * @return {Node} The merror node.
    */
   protected formatError(err: TexError): MmlNode {
```

Comment:
Should that be `public`?

Reply:
No, this is the function that is internal to the TeX input jax.  It calls the format function in the options list, and that one calls the (public) `defaultFormatError()` by default.

Since this protected `formatError()` is only called from the `compile()` function (when an error is thrown), I suppose its contents could be moved to there, and then rename `defaultFormatError()` as (public) `formatError()` and reduce the need for an extra function.  I should have through too that originally.